scripts/leapfrog/playbooks/logstash-start.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Start logstash serivce
+  hosts: logstash_all
+  user: root
+  tasks:
+    - name: Service start
```

Comment:
Is this not covered by the elk upgrade process?

Reply:
At this point logstash has been already upgraded. The reason to stop and restart logstash is elasticsearch upgrade won't need more coming log when the re-indexing is going. So try not to forever hanging re-indexing process, we should turn off logstash temporarily and give it back when elasticsearch upgrade done.